Wordle.Api/Wordle.Api/Program.cs

```diff
@@ -0,0 +1,44 @@
+using Wordle.Api.Services;
+
+var AllOrigins = "AllOrigins";
+
+var builder = WebApplication.CreateBuilder(args);
+

```

Comment:
I believe you need to have a connection string here as well as to add Db context for each model.  At least for the localDb implementation.

Reply:
Can you expand on where you mean by here?